tailwarden/komiser
# feat: fetch DynamoDB costs for RCUs and WCUs

## Problem
We have no costs on DynamoDB resources

## Solution

Fetch costs of DynamoDB resources

## Changes Made

- Extend `aws:dynamodb:tables.go` to fetch RCUs and WCUs through table details
- Add test for util function

## How to Test

Util can be tested with 
```shell
go test ./providers/aws/dynamodb/

// should print
// ok  	github.com/tailwarden/komiser/providers/aws/dynamodb	0.290s
```

Otherwise start komiser locally and see if DynamoDB costs are being fetched properly.

## Screenshot

![Screenshot 2023-10-19 at 15 58 02](https://github.com/tailwarden/komiser/assets/320272/1cabdeee-4e80-4b7d-a5dd-b9454ef185ee)

## Notes

First time doing Go. Feel free to give any tips on what to improve!

## Checklist
